### PR TITLE
W 11229183 exchange update doc impacted by conformance status jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by bjoson-mulesoft — DISMISSED** [review]

File: modules/ROOT/pages/asset-details.adoc [hunk 1]
```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
 This is similar to using a program such as Postman to specify custom HTTP headers and test each API's method using the API's endpoint.
 . To connect the API to one of your applications and use the API to send and receive data, click *Request access*.
 
+== View Conformance Status 
+
+You can view governance conformance status in Exchange for versions of API definitions. The status is one of the following:
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/asset-details.adoc:79]: What are the supported asset types for which conformance status is visible? "API definitions" is a little unclear to me - can we call the specific asset types out upfront?

File: modules/ROOT/pages/asset-details.adoc [hunk 2]
```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
...
+ 
+* A conformance badge is displayed for a selected version of an API.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/asset-details.adoc:87]: while the above sentence says the status can be "one of the following", the text following that is not enumerating the possible status values such as [conformant, not conformant, not validated] etc.. The possible status values need to be documented here with an explanation as to what the value means and in what cases a certain value is present

File: modules/ROOT/pages/asset-details.adoc [hunk 3]
```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
...
+* In the *Manage Versions* page, a *Conformance* column shows conformance status for each version of the API. 
+* A *Conformance Status* page appears for all REST APIs and AsyncAPIs. If a selected version of an API has been validated against governance rulesets, the conformance status page gives conformance information and gives options to fix nonconformant versions.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/asset-details.adoc:89]: Can we call "Rest APIs and Async APIs" out above where we say "API definitions"

File: modules/ROOT/pages/asset-details.adoc [hunk 4]
```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
...
+
+
+To view conformance status:
+
+. Select an API with governance rulesets.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/asset-details.adoc:93]: We should document how a user can find such an API (an API with governance rulesets) - if there is a way to discover those APIs

File: modules/ROOT/pages/asset-details.adoc [hunk 5]
```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
...
+. Review the conformance badge.
+. To view conformance by version, click *Manage Versions*.
+. Review the *Conformance* column, which shows the governance conformance status for each validated version.
++
+. To view conformance details, select *Conformance Status* in the left navigation bar. From the conformance status page, you can:
++
+* Select the version of the API that you want to view.
+* View the lists of centralized governance rulesets and additional rulesets against which the API has been validated.
+** If there are more rulesets than are shown on the initial page, click the *Show All* link to show the full list.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/asset-details.adoc:102]: A screenshot of the conformance status page will be really helpful here.

**Review by rithishapadmanabh — COMMENTED** [review]
Left a few comments. The first 3 are more critical to address. 4 & 5 would be good to have.

File: modules/ROOT/pages/asset-details.adoc (hunk 4 above)
Comment (JennyHajee) [reply modules/ROOT/pages/asset-details.adoc:93]: In a later release, users will be able to search by conformance in Exchange.

File: modules/ROOT/pages/asset-details.adoc (hunk 5 above)
Comment (JennyHajee) [reply modules/ROOT/pages/asset-details.adoc:102]: I will consider adding a screen capture before the instructions after GA.

**Review by bjoson-mulesoft — APPROVED** [review]

**Comment by bjoson-mulesoft** [comment]
Merged on behalf of Jenny because the release finally went live after hours her time.

**Pull request merged**